NitroxClient/Communication/Packets/Processors/JoinQueueInfoProcessor.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxModel.Packets;
+
+namespace NitroxClient.Communication.Packets.Processors;
+
+public class JoinQueueInfoProcessor : ClientPacketProcessor<JoinQueueInfo>
+{
+    public override void Process(JoinQueueInfo packet)
+    {
+        Log.InGame($"You are at position #{packet.Position} in the queue.");
+
+        if (packet.ShowMaximumWait)
+        {
+            Log.InGame($"The maximum wait time per person is {MillisToMinutes(packet.Timeout)} minutes.");
+        }
+    }
+
+    private static string MillisToMinutes(int milliseconds)
+    {
+        double minutes = milliseconds / 60000.0;
+        return Math.Round(minutes, 1).ToString();
```

Comment:
I find that this could be misleading, why would we not just have it in seconds or a combination of both. If the milliseconds came out to 20 seconds we would round it to 0.3 which would require the user to calculate what this actually means